subworkflows/local/input_check.nf

```diff
@@ -31,21 +31,30 @@ def create_spaceranger_channels(LinkedHashMap row) {
     def meta = [:]
     meta.id = row.sample
 
-    def array = []
+    def raw_meta = []
     if (!file(row.fastq_dir).exists()) {
-        exit 1, "ERROR: Please check input samplesheet -> fastq_dir directory does not exist!\n${row.fastq_1}"
+        exit 1, "ERROR: Please check input samplesheet -> fastq_dir directory does not exist!\n${row.fastq_dir}"
     }
     if (!file(row.tissue_hires_image).exists()) {
-        exit 1, "ERROR: Please check input samplesheet -> tissue_hires_image file does not exist!\n${row.fastq_1}"
+        exit 1, "ERROR: Please check input samplesheet -> tissue_hires_image file does not exist!\n${row.tissue_hires_image}"
     }
-    array = [
+    if ( row.manual_alignment.isEmpty() ) {
+        manual_alignment = file ( "EMPTY_ALIGNMENT" )
```

Comment:
It's possible to speficy an empty list (`[]`) as null token. I think it was at some point decided this is the way to go over `EMPTY_XXX`, see https://nf-co.re/docs/contributing/modules#inputoutput-options

Reply:
Thanks, I wasn't aware of this new practice!